app/models/survey/questions_type.rb

```diff
@@ -1,27 +1,27 @@
 class Survey::QuestionsType
-  @@questions_types = {:general => 1}
-  
+  @@questions_types = { multiple_choice: 2, free_response: 9, multi_select: 1 }
```

Comment:
Right, but I'm just asking if nothing else needs to be in this gem. The question types aren't used internally at all today?